.github/workflows/release.yml

```
@@ -9,19 +9,99 @@ on:
+
+  deploy-blue:
+    name: 🔵 Deploy to blue
+    runs-on: ubuntu-latest
+    needs: [build, deploy-alpha]
+    timeout-minutes: 30
+    environment: prod
+    steps:
+      - env:
+          DOCKER_REPO: ${{ vars.DOCKERHUB_REPO }}
+          DOCKER_USERNAME: ${{ vars.DOCKERHUB_USERNAME }}
+          IMAGE_ID: ${{ needs.build.ouputs.IMAGE_ID }}
```

actionlint (1.7.7)

property "ouputs" is not defined in object type {outputs: {image_id: string}; result: string}

(expression)